catroid/src/org/catrobat/catroid/formulaeditor/FormulaElement.java

```diff
@@ -354,6 +354,12 @@ private Object interpretFunction(Functions function, Sprite sprite) {
 				return doubleValueOfLeftChild == null ? 0d : java.lang.Math.toDegrees(Math.atan(doubleValueOfLeftChild));
 			case EXP:
 				return doubleValueOfLeftChild == null ? 0d : java.lang.Math.exp(doubleValueOfLeftChild);
+			case POWER:
+				try {
+					return java.lang.Math.pow(doubleValueOfLeftChild, doubleValueOfRightChild);
+				} catch (Exception e) {
+					return 0d;
+				}
```

Comment:
To be consistent with all other cases, use `(doubleValueOfLeftChild == null || doubleValueOfRightChild == null) ? 0d :..` as well.

If there aren't unit tests, please create some.